status-im/status-react
# Implement dummy action drawers for community chats

Partially implements https://github.com/status-im/status-mobile/issues/16178

### Summary

This PR implements the skeleton for all community channel drawer actions. It is not concerned with the actual implementation of the actions, since that would grow the scope of the PR too much. They will be done piecemeal, by whomever wants/can pick the task in the future.

### Notes

I also updated the actions drawer component to accept a text aligned to the right (this will be necessary to display the number of pinned messages in the action drawer). In the Design System, the text is always hidden, and that's my best guess as to why we didn't implement it. I talked to designers, and they won't update the Design System [Drawer action component](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=1931-31188&t=hOK17fADEXTlnXPY-0) for now (Figma reasons...), but we agreed the reference in [Figma > Communities for Mobile](https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=5483-193285&t=vOTiuhxD87zhmK2T-0) is good enough.

<img src="https://github.com/status-im/status-mobile/assets/46027/9f9a4471-c166-4858-a7a1-d90da73521f1" width="400" />

### Steps to test

1. Create or open a community.
2. Long press in any community chat. If it's locked you'll see less actions (as in Figma), otherwise you'll see the full list of actions.
3. Open a community chat and verify the list of options. When the chat is active they're slightly different.

status: ready
